src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -91,34 +97,40 @@ public class WebScriptHostManager : ScriptHostManager
 
             secretsRepositoryFactory = secretsRepositoryFactory ?? new DefaultSecretsRepositoryFactory();
             var secretsRepository = secretsRepositoryFactory.Create(settingsManager, webHostSettings, config);
-            _secretManager = secretManagerFactory.Create(settingsManager, config.TraceWriter.WithDefaults(ScriptConstants.TraceSourceSecretManagement), config.HostConfig.LoggerFactory, secretsRepository);
```

Comment:
The logger factory being used here before I added the new param was actually null, meaning SecretManager wasn't logging anything.